### PR TITLE
fix service crash

### PR DESCRIPTION
**Pull request by czpmango**
#### What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### What problem does this PR slove?
Issue(s) number: https://github.com/vesoft-inc/nebula/issues/3609

Description:
Service crash when exit.
The crash is caused by the glog lock variable (static variable called `vmodule_lock`) being destructed before thriftServer.

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Perfomance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> Fixed the bug

### CODE REVIEW

**Review by yixinglu — DISMISSED** [review]

File: src/daemons/GraphDaemon.cpp [hunk 1]
```diff
@@ -153,15 +151,15 @@ int main(int argc, char *argv[]) {
   }
   LOG(INFO) << "Number of worker threads: " << FLAGS_num_worker_threads;
 
+  std::unique_ptr<nebula::graph::GraphServer> gServer =
```
Comment (jievince) [inline src/daemons/GraphDaemon.cpp:154]: auto

File: src/daemons/MetaDaemon.cpp [hunk 2]
```diff
@@ -215,14 +216,13 @@ int main(int argc, char* argv[]) {
       std::make_shared<nebula::meta::MetaServiceHandler>(gKVStore.get(), metaClusterId());
   LOG(INFO) << "The meta daemon start on " << localhost;
   try {
-    gServer = std::make_unique<apache::thrift::ThriftServer>();
-    gServer->setPort(FLAGS_port);
-    gServer->setIdleTimeout(std::chrono::seconds(0));  // No idle timeout on client connection
-    gServer->setInterface(std::move(handler));
+    gMetaServer->setPort(FLAGS_port);
+    gMetaServer->setIdleTimeout(std::chrono::seconds(0));  // No idle timeout on client connection
```
Comment (jievince) [inline src/daemons/MetaDaemon.cpp:220]: Unify the name.

File: src/daemons/GraphDaemon.cpp (hunk 1 above)
Comment (czpmango) [reply src/daemons/GraphDaemon.cpp]: ACK.

File: src/daemons/MetaDaemon.cpp (hunk 2 above)
Comment (czpmango) [reply src/daemons/MetaDaemon.cpp]: ACK.

**Review by Aiee — APPROVED** [review]

**Review by SuperYoko — APPROVED** [review]

**Review by SuperYoko — APPROVED** [review]
Don't forget standalone daemon./Maybe I'll do the sync.

**Review by yixinglu — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Pull request merged**